lib/hammer_cli_katello/module_stream.rb

```diff
@@ -0,0 +1,55 @@
+module HammerCLIKatello
+  class ModuleStreamCommand < HammerCLIKatello::Command
+    resource :module_streams
+
+    class ListCommand < HammerCLIKatello::ListCommand
+      output do
+        field :id, _("ID")
+        field :name, _("Module Stream Name")
+        field :stream, _("Stream")
+        field :uuid, _("UUID")
+        field :version, _("Version")
+        field :arch, _("Architecture")
+        field :context, _("Context")
+      end
+
+      build_options
```

Comment:
```sh
$ hammer module-stream list --content-view-version-id 1
undefined method `module_streams' for #<Katello::ContentViewVersion:0x00007ff1a3cf1060>
Did you mean?  model_name
```

Make sure to check `hammer module-stream [subcommand] -h` and test all those options.